### DISCUSSION TITLE
quicsample error: Shut down by transport, 0x8

### DISCUSSION THREAD

**Discussion by thiennguu** [microsoft/msquic#4216] [General]
I have successfully built msquic and the samples, and I have also created a certificate using the following command: openssl req -nodes -new -x509 -keyout server.key -out server.cert. 
However, when I run the quicsample application, it encounters the following error: 

[conn][0x130812a00] Connecting...
[conn][0x130812a00] Shut down by transport, 0x8
[conn][0x130812a00] All done. 

Please show me how to fix this error.

**Comment by nibanks** [comment]
Please take a look at the [Trouble Shooting Guide](https://github.com/microsoft/msquic/blob/main/docs/TSG.md). Likely you will have to grab some logs to debug futher.

**Comment by imorlxs** [comment]
For someone looking that in the future, this 0x8 code means that the address is incorrect or unreachable. Check the -target parameter
